### ISSUE TITLE
Problem bulding

### ISSUE THREAD

**Issue by cledev** [dimkr/szl#10]
After: sudo apt-get install git gcc meson zlib1g-dev libcurl4-gnutls-dev libarchive-dev libssl-dev libffi-dev
After: git clone --recursive https://github.com/dimkr/szl
After: cd szl

:~/szl$ meson build
Error during basic setup:

[Errno 2] No such file or directory: '/home/jake/szl/build'
jake@rexcore2:~/szl$ mkdir build
jake@rexcore2:~/szl$ meson build
The Meson build system
Version: 0.29.0
Source dir: /home/jake/szl
Build dir: /home/jake/szl/build
Build type: native build
Build machine cpu family: x86_64
Build machine cpu: x86_64
Project name: szl
Native c compiler: cc (gcc 5.4.0-6ubuntu1)

Meson encountered an error in file src/meson.build, line 23, column 0:
Unknown function "join_paths".

=====

I am unfamiliar with meson.  Can you see anything I'm doing wrong?

**Comment by cledev** [comment]
For posterity:

The default meson installed with apt-get is too old on Ubuntu 16.04.  The latest version of meson can be installed with pip3, and that works fine.

There is another issue, due to installing to /usr/local/lib/x86_64-linux-gnu/.  One is required to add this path somewhere in /etc/ld.so.conf.d and then run sudo ldconfig afterword.  Once that's all done szlsh does start.  I'm so excited to use it, it looks fantastic.
